### ISSUE TITLE
Update Nuitka doc to account for upstream & Arcade changes

### ISSUE THREAD

**Issue by pushfoo** [pythonarcade/arcade#2270]
### What documentation needs to change? 

Notes on compiling with Nuitka.

### Where is it located?

https://github.com/pythonarcade/arcade/blob/bf5b50af92cbdc7eb63fb1bc7e1fa6d8fa845d0c/doc/tutorials/compiling_with_nuitka/index.rst

### What is wrong with it? How can it be improved?

Quoting @DragonMoffon on Discord:

> Plus at some point when we are improving the nuitka docs please make sure to add the line
> Experimental features of arcade will not be available unless you do `--include-module=acade.experimental`

It seems like we need to add more modules to exclusion / notes on why.

| Module to consider | Currently excluded? | Reason for exclusion
|---------------------------|----------------------------|----------------------------------------------------|
| `experimental`         | Yes                           | Often broken, so most won't use this     |
| `examples`            | Yes                           | The point of examples is to be runnable source |
| `future` | No                    | Unstable like `experimental`, even if a little better |

**Comment by ronnydoan12345** [comment]
Hi! I’d like to work on this documentation update if no one else is currently working on it.

**Comment by Cleptomania** [comment]
I don't believe anyone is actively working on it right now, so feel free to take a run at it.

**Comment by ronnydoan12345** [comment]
Hi, I’ve submitted my first PR!  Let me know if I need to change anything or if there is any feedback !

PR: https://github.com/pythonarcade/arcade/pull/2831
